src/patternfly/components/Radio/examples/radio-disabled-example.hbs

```diff
@@ -0,0 +1,4 @@
+{{#> radio}}
+  {{#> radio-input radio-input--attribute='id="radio4" name="exampleRadio5" disabled'}}{{/radio-input}}
+  {{#> radio-label radio-label--modifier="pf-m-disabled" radio-label--attribute='for="radio4"'}}Radio 4 disabled{{/radio-label}}
+{{/radio}}
```

Comment:
Can you add a checked variation in here? I think it's OK to have both in one example since they aren't interactive.